### DISCUSSION TITLE
matc cannot be opened

### DISCUSSION THREAD

**Discussion by khyati18** [google/filament#6684] [Q&A]
I am getting this error on running matc from Apple M1 Pro. Is there is way around ?

<img width="1289" alt="Screenshot 2023-03-27 at 11 07 58 AM" src="https://user-images.githubusercontent.com/43890805/227850229-85de24d4-81cb-431d-9d00-f008ef1015fb.png">

**Comment by romainguy** [comment]
Right click on it and choose « Open » to get rid of this macOS warning.

**Reply by khyati18** [reply]
It throws the same error still. @romainguy

**Reply by khyati18** [reply]
`sudo spctl –master-disable`

This worked.
